app/models/db/user.py

```diff
@@ -66,7 +67,7 @@ async def update_user_data(self, user_tg):
             await self.save()
 
     @classmethod
-    async def get_or_create_from_tg_user(cls, user_tg: types.User):
+    async def get_or_create_from_tg_user(cls, user_tg: types.User | dto.TargetUser):
```

Comment:
Such typehint will not allow start bot on old python versions, such as 3.9-3.7

Reply:
it's ok. in near future i will drop support of 3.10 python too. It is application and not library, we need to support only one version of python